cli/azd/pkg/exec/util_test.go

```diff
@@ -144,7 +144,7 @@ func TestError(t *testing.T) {
 
 	var exitErr *ExitError
 	require.ErrorAs(t, err, &exitErr)
-	require.Contains(t, exitErr.Error(), "exit code: 2, stdout:")
+	require.ErrorContains(t, exitErr, "exit status 2, stdout:")
```

Comment:
This was one part I left in when I did the error code refactoring work, because:
1. Exit code seems like the more intuitive term over exit status.
2. We want a greppable error message that shows that a tool error occurred. This may be useful for our support engineers. I was explaining to current engineers this way: "If there is an 'exit code' in the error text, it means the failure happened in a third-party tool"

Reverting back to "exit code: 2" gets us number 1, but curious how we can solve number 2, given that we're expanding to include non-exit code errors. Maybe it sits at a higher level, where we do "running tool <x>"? or the likes?

Reply:
Thanks, the context here is helpful.  I will go back to making the implementation of our ExitError use "exit code" and not "exit status" when that is the reason for exit.  The grepabilty argument makes sense to me, I'll include a note in a comment.

Reply:
@weikanglim made a change here, let me know what you think.